apps/system/style/notifications/notifications.css

```diff
@@ -102,7 +87,7 @@
 
 #notification-toaster.displayed {
   transform: translateY(0);
-  transition: transform .2s ease 1.2s, background-color 0s ease 1.2s, visibility 0s ease 1.2s;
+  transition: transform .2s ease .4s, background-color 0s ease, visibility 0s ease;
```

Comment:
Otherwise we don't see the content of the notification until the end.

Even with the change we still have an issue, I think it's a Gecko issue as I see the same when I open the utility tray.